NMRangeSlider/NMRangeSlider.h

```diff
@@ -53,6 +53,9 @@
 // minimum value for right thumb
 @property(assign, nonatomic) float upperMinimumValue;
 
+// track height
+@property(assign, nonatomic) float thickness;
```

Comment:
Is thickness a float or an int? I would expect an int, what happens when passing in a float?